input/ig.xml

```diff
@@ -398,6 +398,14 @@
 			<description value="Active prescription for Reaptan 10 mg/10 mg (perindopril arginine/amlodipine) tablet, 10."/>
 			<exampleCanonical value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-medicationrequest"/>
 		</resource>
```

Comment:
These comments are added on this file so that the conversation can be tracked and closed. They are not to change this file.
1. Missing cleanup on condition-ckd. This doesn't match AU Core, it's one of the american examples - if it is to be retained please adjust to be AU Core e.g. remove the additional meta values.

Reply:
1. Done
2. Removed & replaced with Ronny Irvine SA persona.
3. Removed example - it does not offer anything unique to the IG
4. Removed verificationStatus from two examples; inv ait-1 fails when both clinicalStatus&verificationStatus are absent
5. Done
6. Done
7. Done
8. Done - added to Procedure/stentinsertion